src/execution_plan/ops/op_create.c

```diff
@@ -9,6 +9,29 @@
 #include "../../schema/schema.h"
 #include <assert.h>
 
+// Resolve the properties specified in the query into constant values.
+PendingProperties *_ConvertPropertyMap(Record r, const PropertyMap *map) {
+	PendingProperties *converted = rm_malloc(sizeof(PendingProperties));
+	converted->property_count = map->property_count;
+	converted->keys = map->keys;
```

Comment:
How about instead of holding keys as `const char **` we use an array of attribute ids?
should save us some time when committing the properties.

Reply:
:+1: